pipeline/unified/channel/sign-release-package-linux.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
                         }
                     ]
 
-          - script: 'rename "s/\.AppImage$/.sig/" $(System.DefaultWorkingDirectory)/detachedSignature/*.AppImage'
+          - script: 'sudo apt-get install rename; rename "s/\.AppImage$/.sig/" $(System.DefaultWorkingDirectory)/detachedSignature/*.AppImage'
```

Comment:
I think you can probably use bash to avoid installing a dependency for this with something like:

```yaml
- bash: 'for file in *.AppImage; do mv -- "${file}" "${file/%AppImage/sig}"; done'
```

If we do need to install a dependency, probably best to do it in a separate step to make it easier to separate apt-get failures vs rename failures in build failure logs/analytics